ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -187,11 +187,34 @@ open class BadgeView: UIView {
 
     @objc open var isSelected: Bool = false {
         didSet {
-            updateBackgroundColor()
-            updateLabelTextColor()
+            updateColors()
             updateAccessibility()
         }
     }
+    
+    open var labelTextColor: UIColor? {
```

Comment:
If the goal is to enable customization of colors, why are we not providing a custom property for the disabled color?

Reply:
This is a good idea, I have updated to allow client to customize disabled background color and text color

Reply:
by having this open variable, I think that I would always get the labelTextColor of my badge view from a client side of view. Not just available when I set some sort of custom color

Reply:
Thanks Harrie for catching this. I have moved default values in property getter/setters. This would make sure the values are always consistently returned what's visible in the UI